packages/wallet/src/redux/adjudicator-state/state.ts

```diff
@@ -0,0 +1,13 @@
+import { Commitment } from 'fmg-core/lib/commitment';
+
+export interface AdjudicatorState {
+  [channelId: string]: AdjudicatorChannelState;
+}
+export interface AdjudicatorChannelState {
+  balance: string;
+  concluded: boolean;
+  challenge?: {
+    expiresAt: number;
+    challengeCommitment: Commitment;
+  };
+}
```

Comment:
Would be nice to have some helper functions here that could be used in the reducer e.g.

* For the `AdjudicatorState`:
    * `getChannel`
    * `setChannel`
* For the `AdjudicatorChannelState`:
    * `clearChallenge`
    * `setChallenge`
    * `setBalance`
    * `markAsFinalized`
    * etc.